src/main/java/com/philderbeast/prisonpicks/PickCommands.java

```diff
@@ -41,6 +46,21 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                     return true;
                 }
 
+                if (args[0].equals("debug") && (sender).hasPermission("picks.debug")) 
```

Comment:
NOOOOOOOOOOOOOOO this should be left to be turned on in config not as a random command.

DEBUG should NEVER be turned on a live server.